styles.css

```diff
@@ -0,0 +1,250 @@
+* {
+    padding:0;
+    margin:0;
+    
+}
+body {
+}
+header {
+    width: 100%;
+    height:225px;
+    background-color:navy;
+
+}
+.header-holder {
+    width:90%;
+    margin: 0px auto;
+    display:flex;
+    justify-content: space-between;
+}
+
+.left-side-header {
+   margin-top: 3%;
+   color:lightgreen;
+}
+.left-side-header img {
+    width:75%;
+}
+.hidden-header {
+    font-size: 1px;
+    visibility:hidden;
+}
+.left-side-header p {
+    font-size:1.5em;
+    font-style:italic;
+}
+.right-side-header {
+    color:white;
+    margin-top: 3%;
+}
+.shopping-cart {
+    width:20px;
+    padding-right: 2px;
+}
+.right-side-header button{
+    margin-top: 3%;
+    width:100%;
+}
+nav {
+    width:100%;
+    height:80px;
+    background-color:beige;
+
+}
+ul.header-nav-list-holder {
+    margin-left:8%;
+    height:80px;
+    line-height:80px;
+    margin-right:5px;
+    display:flex;
+    flex-direction:row;
+    list-style-type: none;
+}
+a {
+    color:navy !important;
+    text-decoration:none;
+}
+a.needs-slash::after {
+    content: "/";
+    padding-left:10px;
+    margin-right:-5px;
+}
+a.needs-slash2::after {
+    content: "/";
+    padding-left:5px;
+    padding-right: 5px;
+}
+li.header-nav-list-item {
+    
+    margin-right:20px;
+}
+.dogs-list-header {
+    margin-left: 7.5%;
+    margin-top: 2%;
+}
+.page-header {
+    margin-left: 7.5%;
+    margin-top:2%;
+    margin-bottom:2%;
+}
+/*
+
+#wrapper {
+    width:85%;
+    height:1000px;
+    margin:0px auto;
+    background-color:white;
+    
+}
+*/
+.grid-item-container {
+    width:85%;
+    margin: 0px auto;
+    display:grid;
+    column-gap: 6.67%;
+    grid-template-columns: 20% 20% 20% 20%;
+    grid-template-rows: auto;
+ 
+}
+.initial-box {
+    margin-top:20px;
+    grid-column: 1/ span 4;
+    display:flex;
+    flex-direction:row;
+    
+}
+.sub-initial-box {
+    margin-left: 6.67%;
+}
+h2.adopt-primary-header {
+    margin-bottom: 10px;
+}
+.intro-para {
+    line-height:35px;
+}
+.adoption-pic {
+    width:100%;
+}
+.initial-box img {
```

Comment:
This image doesn't really scale well going narrower. There are a few ways you could do this. maybe image height fit-content?  Or.     width: 30%;  max-height: 100%; object-fit: contain; There are a few ways you could handle this